homeassistant/components/meraki/device_tracker.py

```diff
@@ -27,12 +31,15 @@
 )
 
 
-async def async_setup_scanner(hass, config, async_see, discovery_info=None):
+async def async_setup_scanner(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_see: Callable[..., Awaitable[None]],
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
```

Comment:
The return value typing is incorrect. It should be True on success. We'll need to update the code too:

https://github.com/home-assistant/core/blob/c6aaa240272b832837ff1158cd4db9df0e07f34b/homeassistant/components/device_tracker/legacy.py#L251

https://github.com/home-assistant/core/blob/c6aaa240272b832837ff1158cd4db9df0e07f34b/homeassistant/components/device_tracker/legacy.py#L270

Reply:
Aaargh :(
Stupid tracker platform